### DISCUSSION TITLE
Bot Composer - Conversation handover to human

### DISCUSSION THREAD

**Discussion by am8492** [composer/composer#9673] [Q&A / Support] [answered]
Hello, 

working on conversation handover to human via Teams channel. I'm missing any complex manual how to configure. 

Using Bot composer v1.2 and I can find only docs in "Microsoft Health Bot Service" but if I'm not mistaken, thats different technology right?
 
Even though there is "Handoff" activity in composer, don't know how to configure. I use composer, since my coding skills are very basic. Does anybody have similiar problem?

Thank you.

![image](https://user-images.githubusercontent.com/70146191/107209006-95c31100-6a02-11eb-9fcc-fdb6a43e802f.png)

**Comment by xabbuh** [comment, marked as answer]
Are you sure that your question is related to the PHP dependency manager Composer and not about https://docs.microsoft.com/en-us/composer/introduction?
